geomesa-kafka/geomesa-kafka-confluent/src/test/scala/org/locationtech/geomesa/kafka/confluent/ConfluentKafkaDataStoreTest.scala

```diff
@@ -89,148 +133,192 @@ class ConfluentKafkaDataStoreTest extends Specification {
       }
     }
 
-    "use the schema overrides config if available" in new ConfluentKafkaTestContext {
-      private val id = "1"
-      private val record = new GenericData.Record(schema2_NoGeoMesa)
-      private val expectedGeom = generatePoint(-12.2d, 120.7d)
-      record.put("shape", ByteBuffer.wrap(WKBUtils.write(expectedGeom)))
-      record.put("date", 1639145281285L)
+    "produce messages based on simple features" in {
```

Comment:
See https://github.com/fdfea/geomesa/blob/GEOMESA-3167/geomesa-kafka/geomesa-kafka-confluent/src/test/scala/org/locationtech/geomesa/kafka/confluent/ConfluentKafkaDataStoreTest.scala#L277 for example

Reply:
wilco, i think the current test is doing that, just using the geomesa confluent serialization, but it would be good to have an explicit confluent read too.